velox/common/memory/MemoryAllocator.h

```diff
@@ -311,6 +311,47 @@ class MemoryAllocator : public std::enable_shared_from_this<MemoryAllocator> {
   /// multiple of 'alignmentBytes'.
   static void alignmentCheck(uint64_t allocateBytes, uint16_t alignmentBytes);
 
+  /// Causes 'failure' to occur in memory allocation calls. This is a test-only
+  /// function for validating otherwise unreachable error paths. If 'persistent'
+  /// is false, then we only inject failure once in the next call. Otherwise, we
+  /// keep injecting failures until next 'testingClearFailureInjection' call.
+  enum class Failure {
```

Comment:
Does this enum represent an injected failure? If so, consider renaming to InjectedFailure or TestFailure to clarify that this applies to testing code only.

Reply:
Changed to InjectedFailure